crates/bevy_app/src/app_builder.rs

```diff
@@ -145,6 +164,10 @@ impl AppBuilder {
         self
     }
 
+    /// Fetches the [`Stage`] of type `T` marked with `label` from the [`Schedule`], then
+    /// executes `func` using the stage as parameter.
```

Comment:
I am really unclear from these docs what `func` does, or why you might use this method.